rocky/katalogus/views/katalogus_settings.py

```diff
@@ -51,11 +51,10 @@ def post(self, request, *args, **kwargs):
 
 
 @class_view_decorator(otp_required)
-class KATalogusSettingsListView(OrganizationPermissionRequiredMixin, OrganizationView, FormView, ListView):
+class KATalogusSettingsListView(OrganizationPermissionRequiredMixin, OrganizationView, FormView):
```

Comment:
Maybe also call the view `KATalogusSettingsView` or something like that because it isn't a ListView?